src/css/colors.css

```diff
@@ -0,0 +1,3 @@
+.bg-primary-dark {
+  background-color: color-mod(#7048e8 blackness(20%));
```

Comment:
create variables for all our colors to centralize them in one place.

Reply:
other color classes specified in tailwind.config and built. this is the only one that needed external function

Reply:
still, it makes more sense to start this now (store our colors in one place not directly in the the selectors/css property)

our next step is to learn how can we create one color and dynamically generate the shades of this color (alignment with Mostafa could help here) and store all or variable in one place, not sure how can we import/export color variables from tailwind config to css files and the other way around, but we should learn how to do so.